src/DevChatter.Bot.Core/IChatUserCollection.cs

```diff
@@ -7,12 +7,12 @@ namespace DevChatter.Bot.Core
     public interface IChatUserCollection
     {
         ChatUser GetOrCreateChatUser(string displayName, ChatUser chatUser = null);
-        bool NeedToWatchUser(string displayName);
+        bool NeedToWatchUser(string displayName, ChatUser chatUser);
```

Comment:
No need to pass both `displayName` and `chatUser`, because the former is on the latter.